test/test_sync_subscriptions.cpp

```diff
@@ -1092,4 +1092,34 @@ TEST(Sync_MutableSubscriptionSetOperations)
     }
 }
 
+TEST(Sync_MutableSubscriptionReleasesReadLock)
+{
+    SHARED_GROUP_TEST_PATH(sub_store_path);
+    SubscriptionStoreFixture fixture(sub_store_path);
+    auto store = SubscriptionStore::create(fixture.db);
+
+    auto read_tr = fixture.db->start_read();
+    Query query_a(read_tr->get_table("class_a"));
+    query_a.greater_equal(fixture.bar_col, int64_t(1));
+
+    size_t num_versions_before_subscription = fixture.db->get_number_of_versions();
```

Comment:
Windows build failures are because get_number_of_versions is a int64_t, not size_t.